eos-remove-old-flatpak-extension.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Remove old Flatpak unmanaged extensions directory
+DefaultDependencies=false
+Conflicts=shutdown.target
+Before=systemd-tmpfiles-setup.service
+After=local-fs.target
+Wants=local-fs.target
+
+# Only run on ARM systems where the old directory exists
+ConditionArchitecture=arm
+ConditionPathExists=/var/lib/flatpak/extension/org.freedesktop.Platform.GL.host
+ConditionPathIsDirectory=/var/lib/flatpak/extension
```

Comment:
"Except for ConditionPathIsSymbolicLink=, all path checks follow symlinks."

I guess this is always before tmpfiles runs, but I'd feel a little more confident if we added a ConditionPathIsSymbolicLink=!/var/lib/flatpak/extension just so these jobs are idempotent and will can never go in circles doing/undoing each other's work. :)

Reply:
Agreed